pg_search/src/query/mod.rs

```diff
@@ -525,7 +531,7 @@ fn value_to_term(
                     let tantivy_datetime = tantivy::DateTime::from_timestamp_micros(
                         datetime.and_utc().timestamp_micros(),
                     );
-                    Term::from_field_date(field, tantivy_datetime)
+                    Term::from_field_date_for_search(field, tantivy_datetime)
```

Comment:
>    /// Builds a term given a field, and a `DateTime` value to be used in searching the inverted
    /// index.
    /// It truncates the `DateTime` to the precision used in the index